content/cloud/noops-sql-db.md

```diff
@@ -0,0 +1,117 @@
+ 
+> Please note that the private beta is limited in space and all requests cannot be guaranteed. 
+
+## Quotas And Service Limitations For NoOps SQL Database in Fermyon Cloud
+
+*Quotas* 
+* You can have a maximum of 1 NoOps SQL Database in your Cloud account
+* Your NoOps SQL Database can hold up 1 GB of data
+
+*Service Limitations*
+* You must delete your Spin application before you can delete your NoOps SQL Database
+* The `--sqlite` flag not supported on `spin cloud deploy`
+* Fermyon Cloud only supports "default" NoOps SQL Databases at this time
+
+## How To:
+
+### Prepare a NoOps SQL Database
+
+We recommend using the `execute` command in your application's source code to create a table. For example, in this routing function we use the Rust SDK to check if the table exists, and to create it if not:
+
+```rust
+#[http_component]
+fn handle_todo(req: Request) -> anyhow::Result<Response> {
+    let connection = Connection::open_default()?;
+	connection.execute("CREATE TABLE IF NOT EXISTS todos (
+			id INTEGER PRIMARY KEY AUTOINCREMENT,
+			description TEXT NOT NULL,
+			due_date DATE,
+			starred BOOLEAN DEFAULT 0,
+			is_completed BOOLEAN DEFAULT 0
+			);", &[])?;
+<...>
+}
+```
+
+### Manage NoOps SQL Database Lifecycle
+
+A NoOps SQL Database is created when you deploy a Spin application that references a "default" SQLite database in its component manifest. Multiple components in the same Spin application can be given access to the default database like so:
+
+```toml
+# c1 does not have access to the default database
+[component]
+name = "c1"
+
+# c2 can use the default database
+[component]
+name = "c2"
+sqlite_databases = ["default"]
+
+# c3 can use default database as well
+[component]
+name = "c3"
+sqlite_databases = ["default"]
+```
+
+The NoOps SQL database will created when the Spin application is deployed. The data persists Spin application invocations and updates. 
+
+```bash
+$ spin cloud deploy
+Deploying app ..
+Creating database inspirational-pig (default) for app "quickstart"
+```
+
+You can check to see your NoOps SQL database by using the `spin cloud sqlite` command:
+
+>> Fermyon Cloud will automatically generate a random name for your NoOps SQL database.
+
+```bash
+$ spin cloud sqlite list
+inspirational-pig (default) 
+```
+
+Lastly, after [deleting your Spin application](/cloud/delete.md) you can delete your default database with the following command:
+
+```bash
+$ spin cloud sqlite delete inspirational-pig
+inspirational-pig has been deleted
+```
+
+> In the Private Beta, the database is inaccessible once the application has been deleted. However, we separate out the the database deletion step because we anticipate the database being accessible after deletion in a future beta. We enforce NoOps SQL Database deletion to prevent unintentional data sharing between Spin applications. 
+
+## FAQ
```

Comment:
Somewhere (whether here or elsewhere on the page) it ought to explain "if you do a deploy and hit such-and-such an error, it means the app wants a database but the one database slot is already taken up"